manifests/0000_51_olm_02_operator_clusterrole.yaml

```diff
@@ -5,8 +5,11 @@ kind: ClusterRole
 metadata:
     name: cluster-olm-operator
     annotations:
-        include.release.openshift.io/self-managed-high-availability: "true"
-        release.openshift.io/feature-set: TechPreviewNoUpgrade
+      include.release.openshift.io/hypershift: "true"
+      include.release.openshift.io/ibm-cloud-managed: "true"
+      include.release.openshift.io/self-managed-high-availability: "true"
+      include.release.openshift.io/single-node-developer: "true"
```

Comment:
This capability is no longer used for anything, e.g. see openshift/cluster-version-operator#685 and openshift/api#1827.  You probably want to drop it here too, to avoid carrying around dead code.